src/System.Runtime/tests/project.json

```diff
@@ -32,6 +32,11 @@
       "dependencies": {
         "System.Runtime.Serialization.Formatters": "4.3.0-beta-24530-03"
       }
+    },
+    "netcoreapp1.1": {
+      "dependencies": {
+        "System.Runtime.Serialization.Formatters": "4.3.0-beta-24530-03"
```

Comment:
Thanks for the pointer looks like @alexperovich added it to do some BinaryFormatter testing. I'm OK with this for testing purposes.